kmodes/util/tests/test_dissim.py

```diff
@@ -25,6 +26,33 @@ def test_matching_dissim(self):
         b = np.array([['a', 'b', 'c', 'd'], ['d', 'c', 'b', 'a']])
         assert_array_equal(np.array([0, 4]), matching_dissim(a, b))
 
+    def test_jaccard_dissim_binary(self):
+        a = np.array([[0, 1, 1, 0, 1, 1]])
+        b = np.array([[0, 1, 1, 0, 1, 0]])
+        assert_equal(0.75, jaccard_dissim_binary(a, b))
+
+        a = np.array([[0, 1, 1, 0, 1, 1]])
+        b = np.array([[0, np.NaN, 1, 0, 1, 0]])
+        with self.assertRaises(ValueError):
+            jaccard_dissim_binary(a, b)
+
+        # test where values are non binary but also not having np.NaN
+        a = np.array([[0, 1, 2, 0, 1, 2]])
+        b = np.array([[0, 1, 2, 0, 1, 0]])
+        with self.assertRaises(ValueError):
+            jaccard_dissim_binary(a, b)
+
+    def test_jaccard_dissim_label(self):
```

Comment:
Please add tests where dissimilarity is 0 and 1.

Reply:
Just to be sure I should be calculating Jaccard distance instead of Jaccard coefficient right? Have a look here [https://people.revoledu.com/kardi/tutorial/Similarity/Jaccard.html](url)

Reply:
That's correct

Reply:
Yes, I did that.